package.json

```
@@ -19,8 +19,8 @@
     "vercel-build": "./scripts/prismaUtils.js baseline migrate && next build"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "eslint --fix",
-    "*.{json,md,scss}": "prettier --write",
+    "*.{js,jsx,ts,tsx,mdx}": "eslint --fix",
+    "*.{json,md,scss,mdx}": "prettier --write",
```

`eslint --fix` already applies prettier style rules.

```suggestion
    "*.{js,jsx,ts,tsx,mdx}": "eslint --fix",
    "*.{json,md,scss}": "prettier --write",
```